plot.py

```diff
@@ -8,26 +8,30 @@
 sums = data[:,1]/1000
 diffs = data[:,2]/1000
 
-plt.style.use('ggplot')
 
-fig = plt.figure()
-gs = gridspec.GridSpec(2, 1)
+def func():
```

Comment:
Hmm, the function name should be something more expressive. Maybe `plot_all_the_important_income_statistics()`.